packages/protocol/contracts/governance/Proposals.sol

```diff
@@ -89,6 +89,7 @@ library Proposals {
     proposal.timestamp = now;
 
     uint256 dataPosition = 0;
+    delete proposal.transactions;
```

Comment:
why this?

Reply:
storage struct is being reused, although I agree this is some code smell

Reply:
Can we avoid using storage for the hotfix proposal, and instead `make` in memory?

Reply:
yes please

Reply:
keeping the delete (which has no effect on empty Proposals), but guarantees that a Proposal cannot be reused improperly